### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by DiegoBM**
Added both react native projects for #30daysofappwrite

### CODE REVIEW

**Review by lohanidamodar — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -53,6 +53,8 @@ Showcase for projects built during [#30DaysOfAppwrite](https://30days.appwrite.i
 Submit your projects and win some Awesome Appwrite Swag 🤩 
 <!-- Raise a PR and add a link to your project under this section. -->
 * [Title of your Project](http://link_to_your_project)
+* [react-native-appwrite-oauth](https://github.com/DiegoBM/react-native-appwrite-oauth)
```
Comment (christyjacob4) [inline README.md:56]: @DiegoBM  Just a small comment, 
Can we skip the dashes from the project name ? More like
`React Native Appwrite OAuth`

:smiley:

Comment (christyjacob4) [inline README.md:56]:
```suggestion
* [React Native Appwrite OAuth](https://github.com/DiegoBM/react-native-appwrite-oauth)
```

File: README.md [hunk 2]
```diff
@@ -53,6 +53,8 @@ Showcase for projects built during [#30DaysOfAppwrite](https://30days.appwrite.i
...
+* [appwrite-todo-with-react-native](https://github.com/DiegoBM/appwrite-todo-with-react-native)
```
Comment (christyjacob4) [inline README.md:57]:
```suggestion
* [Appwrite Todo With React Native](https://github.com/DiegoBM/appwrite-todo-with-react-native)
```

**Review by christyjacob4 — CHANGES_REQUESTED** [review]
Please address the comments

File: README.md (hunk 1 above)
Comment (DiegoBM) [reply README.md:56]: By all means, anything that makes it look better :)

**Review by christyjacob4 — APPROVED** [review]
Perfect :+1:

**Pull request merged**